### PR TITLE
Install versions defined in Chef environment

### PR DESCRIPTION
**Pull request by chef-delivery**
This Pull Request was opened by Chef Automate user Seth Chisamore

This change requires merging of chef-cookbooks/habitat-build#18

Signed-off-by: Seth Chisamore <schisamo@chef.io>

Ready to merge? View this change in Chef Automate and click the Approve button: https://delivery.chef.co/e/chef/#/organizations/CIA/projects/omnitruck/changes/39e739ca-0f46-4a7d-8426-65b27379a183

### CODE REVIEW

File: .delivery/build-cookbook/attributes/default.rb [hunk 1]
```diff
@@ -2,3 +2,8 @@
 # systemd for habitat's services.
 default['ciainfra']['image_id'] = 'ami-7ac6491a'
 default['ciainfra']['security_groups'] = ['sg-9d9223f9']
+
+default['delivery']['project_apps'] = %w(
```
Comment (wrightp) [inline .delivery/build-cookbook/attributes/default.rb]: How are these attributes used?

Comment (schisamo) [inline .delivery/build-cookbook/attributes/default.rb]: You can read up more about the _Project Applications_ pattern at:
https://docs.chef.io/delivery_truck.html#project-applications

Comment (wrightp) [inline .delivery/build-cookbook/attributes/default.rb]: 👍

**Pull request merged**